### PR TITLE
[TM-1531] add job service url and script

### PR DESCRIPTION
**Pull request by egrojMonroy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "generate:api": "openapi-codegen gen api",
+    "generate:jobService": "openapi-codegen gen jobService",
     "generate:userService": "openapi-codegen gen userService",
     "generate:services": "npm run generate:userService",
```
Comment (roguenet) [inline package.json:21]: please add it to this one too. Also, I should have used `yarn` here, not `npm run`, so please change that as well:
```
"generate:services": "yarn generate:userService && yarn generate:jobService"
```

File: src/generated/v3/jobService/jobServiceFetcher.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
+export type JobServiceFetcherExtraProps = {
```
Comment (roguenet) [inline src/generated/v3/jobService/jobServiceFetcher.ts:1]: See the note in the README about this. This file should be mostly empty. Take a look at userServiceFetch.ts for an example.

Comment (egrojMonroy) [reply src/generated/v3/jobService/jobServiceFetcher.ts]: readed and follow the instructions for this.

File: src/store/apiSlice.ts [hunk 3]
```diff
@@ -53,12 +54,13 @@ type StoreResourceMap<AttributeType> = Record<string, StoreResource<AttributeTyp
...
 // The list of potential resource types. IMPORTANT: When a new resource type is integrated, it must
 // be added to this list.
-export const RESOURCES = ["logins", "organisations", "users"] as const;
+export const RESOURCES = ["logins", "organisations", "users", "delayedJobs"] as const;
 
 type ApiResources = {
   logins: StoreResourceMap<LoginDto>;
   organisations: StoreResourceMap<OrganisationDto>;
   users: StoreResourceMap<UserDto>;
+  delayedJobs: StoreResourceMap<DelayedJobDto>;
```
Comment (roguenet) [inline src/store/apiSlice.ts:63]: :art: a minor thing, but I think keeping this in alphabetical order is sort of nice.

Comment (egrojMonroy) [reply src/store/apiSlice.ts]: sorted

**Review by roguenet — APPROVED** [review]

**Pull request merged**